plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy

```diff
@@ -225,12 +228,46 @@ class GoogleBatchTaskHandler extends TaskHandler implements FusionAwareTask {
             instancePolicyOrTemplate.setInstallGpuDrivers(true)
         }
 
+        if( fusionEnabled() && !disk )
+            disk = new DiskResource(request: '375 GB', type: 'local-ssd')
```

Comment:
If the user has specific, for example, `disk 100.GB` and fusion is enabled, this condition is false and no temporary storage will be accessible to Fusion. @jordeu is this wanted?

Reply:
It is expected that if the user specifies a disk while using Fusion, they will specify the disk that is used by Fusion. That way they can make Fusion use a disk type other than local SSD, if they don't want to be subject to quota around local SSD. I believe that was the original motivation for this PR.

Reply:
Yes, thats wanted. See issue #3776